public/script/listUpdates.json

```
@@ -0,0 +1,247 @@
+[
+{
+    "position":0,
```

```suggestion
    "position":0,
    "HeroImage_url":"ArtistName_Hero_Asset.jpg",
```
Keep name convention through same file type consistency (in this case for .json files), if you start with uppercase, all properties should be in uppercase, if you _ (underscore) for spaces, use it through all properties.